lib/init.js

```diff
@@ -31,6 +31,19 @@ const findElixirProjectPath = async (editorPath) => {
   const editorDir = dirname(editorPath);
   const mixexsPath = find(editorDir, 'mix.exs');
   if (mixexsPath !== null) {
+    const pathArray = mixexsPath.split('/');
+    if (pathArray.length > 3 && pathArray[pathArray.length - 3] === 'apps') {
+      //  Treat this as an umbrella app. This may be wrong -
+      //  If you happen to keep your code in a directory called 'apps'
+      pathArray.splice((pathArray.length - 3), 3);
+      const umbrellaProjectPath = pathArray.join('/');
```

Comment:
Ah, my mistake, from the notation in [the documentation](https://nodejs.org/api/path.html#path_path_join_paths) it looked to accept an array but that appears to not be the case.

Probably the simplest is to just do `pathArray.join(sep)`, since that will use the OS specific separator (and is basically what `path.join` does in the first place).